build.sbt

```diff
@@ -44,7 +45,13 @@ ThisBuild / githubWorkflowAddedJobs ++= Seq(
   )
 )
 
-val docMappingsApiDir = settingKey[String]("Subdirectory in site target directory for API docs")
+def do_configure(project: Project) = project.settings(
+  Seq(
+    Test / scalacOptions -= "-Xfatal-warnings"
+  )
+)
```

Comment:
I don't think we need this?  I removed it and ran `prePR` and didn't get any errors/warnings?

Reply:
it did not work in ci with this on, so maybe prePR does not turn it on?